### DISCUSSION TITLE
Is there a way to disable commands while in PVP Combat?

### DISCUSSION THREAD

**Discussion by ArvickC** [GriefPrevention/GriefPrevention#1069] [Q&A]
The answer to your question may exist in the wiki, go check that out first! **Not on the wiki**

Wiki: https://github.com/TechFortress/GriefPrevention/wiki

Otherwise, if you have a spigot account, you may have better luck posting in the discussion thread so people who follow the thread may see your question. https://www.spigotmc.org/threads/griefprevention.35615/page-311 **Haven't found anything so far**

Still no luck? Really? Then go ahead and ask here.

Please remember to provide as much detail as you can - smart questions are
much more likely to be answered.  See "Asking questions the smart way":
http://www.catb.org/~esr/faqs/smart-questions.html

As an open-source project, we're all volunteers with limited time to spend,
so providing enough info (including any error messages etc) up-front makes
it more likely we can provide you an answer with limited time.

Hello, Its just a very simple question but I couldn't find it in the config (if it is even there) but I wanted to know if there is a way to disable commands that are run by the user, while in PVP Combat?

**Comment by RoboMWM** [comment]
It's in the config...

Now I'm gonna check the wiki.

**Comment by RoboMWM** [comment]
Interesting, it isn't in there. Well then. But it's in the config in the pvp section.

**Comment by ArvickC** [comment]
Could you tell me what the name of it is called... I can't seem to find it? @RoboMWM

**Comment by KJNine** [comment]
`BlockedSlashCommands: /home;/vanish;/spawn;/tpa` on line 80 in the `PvP:` section. Only works for the first part of the command, so blocking something like `/gamemode creative` without blocking `/gamemode survival` isn't possible, however if that really matters there are several other plugins with that functionality.

**Comment by RoboMWM** [comment]
Hmm, what does it do if there's a space? I assumed it did nothing more than playercommandpreprocess event and split into array on `;`

**Comment by Jikoo** [comment]
Spaces result in no blocking whatsoever, it [compares](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java#L470) specifically the [command without arguments](https://github.com/TechFortress/GriefPrevention/blob/master/src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java#L396). Would be a neat addition, but I'd hope to see a system for blocking aliases first.

**Comment by RoboMWM** [comment]
Ah, yea, that handler is geared more for the chat features instead
